### DISCUSSION TITLE
Download/install options (windows)

### DISCUSSION THREAD

**Discussion by rzw** [ransome1/sleek#483] [General]
Regarding windows there are three options to run the program

1. installation via setup (sleek-1.3.1-win-Setup.exe)
2. portable "installation" via zip-archive (sleek-1.3.1-win.zip)
3. standalone exe (sleek-1.3.1-win.exe)

All files are more or less the same size which obviously(?) means that the standalone sleek-1.3.1-win.exe unpacks in volatile memory/RAM, which is why it takes longer to start up the application. Of course there are many applications with standalone exe's. But I haven't seen this as an option besides setup and archive. So I thought I'd mention it here.

**Comment by ransome1** [comment]
@rzw so you're concluding the .exe build is rather unusual and most likely not widely used?

**Reply by rzw** [reply]
I'm sure there's a reason for the option. The geeks will know.

I just wanted to facilitate the choice for people who want a portable version that starts up fast. Please correct me in case my observation is wrong.

**Reply by ransome1** [reply]
I tried the standalone standalone .exe myself and have to say it is really significantly slower than its alternatives. And I think the zip version is as portable as the standalone one. So yeah, I don't see the need for it either.

The reason we have the .exe version is that I don't really need to do anything to create it. It just comes out of the build process, which is done by Electron Builder. As this doesn't cost me neither money nor time, I never really thought about it twice ;)
